README.md

```diff
@@ -1,33 +1,88 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-pages.svg)](https://www.npmjs.com/package/@polymer/iron-pages)
 [![Build status](https://travis-ci.org/PolymerElements/iron-pages.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-pages)
-[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/PolymerElements/iron-pages)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-pages)
 
-_[Demo and API docs](https://elements.polymer-project.org/elements/iron-pages)_
+## &lt;iron-pages&gt;
 
+`iron-pages` is used to select one of its children to show. One use is to cycle
+through a list of children "pages".
 
-## &lt;iron-pages&gt;
+See: [Documentation](https://www.webcomponents.org/element/@polymer/iron-pages),
+  [Demo](https://www.webcomponents.org/element/@polymer/iron-pages/demo/demo/index.html).
+
+## Usage
 
-`iron-pages` is used to select one of its children to show. One use is to cycle through a list of
-children "pages".
+### Installation
+
+```
+npm install --save @polymer/iron-pages
+```
 
-Example:
+### In an HTML file
 
 ```html
-<iron-pages selected="0">
-  <div>One</div>
-  <div>Two</div>
-  <div>Three</div>
-</iron-pages>
-
-<script>
-  document.addEventListener('click', function(e) {
-    var pages = document.querySelector('iron-pages');
-    pages.selectNext();
-  });
-</script>
+<html>
+  <head>
+    <script type="module">
+      import '@polymer/iron-pages/iron-pages.js';
+    </script>
+  </head>
+  <body>
+    <iron-pages selected="0">
+      <div>Page 1</div>
+      <div>Page 2</div>
+      <div>Page 3</div>
+    </iron-pages>
+  </body>
+</html>
 ```
 
-### Notable breaking changes between 1.x and 2.x (hybrid):
+### In a Polymer 3 element
+
+```js
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+
+import '@polymer/iron-pages/iron-pages.js';
+
+class ExampleElement extends PolymerElement {
+  static get template() {
+    return html`
+      <iron-pages selected="0">
+        <div>Page 1</div>
```

Comment:
Probably make these pages 0-indexed so that selected is much easier to associate?